docs/5min-overview.md

```diff
@@ -72,7 +72,8 @@ Find our latest [node release on GitHub][node-release].
 Discover and contribute to our main projects [on GitHub][github]:
 
 - [Full node][full-node]
-- [Mobile Wallet & Desktop Wallet](https://github.com/alephium/alephium-frontend)
+- [Desktop wallet][desktop-wallet]
+- [Mobile wallet][mobile-wallet]
```

Comment:
At the bottom of the file you need to add the correct links to these 2 link references.

```
[desktop-wallet]: https://github.com/alephium/alephium-frontend/apps/desktop-wallet
[mobile-wallet]: https://github.com/alephium/alephium-frontend/apps/mobile-wallet
```